crates/editor/src/scroll.rs

```diff
@@ -136,7 +156,7 @@ pub struct ScrollManager {
     last_autoscroll: Option<(gpui::Point<f32>, f32, f32, AutoscrollStrategy)>,
     show_scrollbars: bool,
     hide_scrollbar_task: Option<Task<()>>,
-    dragging_scrollbar: bool,
+    dragging_scrollbar: (bool, bool),
```

Comment:
Is it `(horizontal, vertical)` or `(vertical, horizontal)`?
We can probably use `AxisPair` here?

Reply:
Thanks for pointing that out, totally forgot to change that one. I initially just used tuples of booleans, but changed them for a new type for that exact reason.